File: app/src/main/java/org/example/product/app/AppServlet.java
package org.example.product.app;

import static java.util.Objects.requireNonNull;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class AppServlet {

    @GetMapping("/")
    public String index() throws IOException {
        new MainModule().run();
        String version = new BufferedReader(
                        new InputStreamReader(requireNonNull(AppServlet.class.getResourceAsStream("/version.txt"))))
                .readLine();
        return "<html><body>" + "App is running... !!!!" + "<br/>Version " + version + "</body></html>";
    }
}

File: app/src/main/kotlin/org/example/product/app/AppServlet.kt
package org.example.product.app

import javax.servlet.annotation.WebServlet
import javax.servlet.http.HttpServlet
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

@WebServlet("/check")
class AppServlet : HttpServlet() {

    public override fun doGet(req: HttpServletRequest, res: HttpServletResponse) {
        res.contentType = "text/html"
        val pw = res.writer
        pw.println("<html><body>")
        pw.println("App is running...")
        pw.println("</body></html>")
        pw.close()
    }
}
